src/ring_theory/ideal/local_ring.lean

```diff
@@ -345,6 +345,16 @@ lemma ker_eq_maximal_ideal [field K] (φ : R →+* K) (hφ : function.surjective
   φ.ker = maximal_ideal R :=
 local_ring.eq_maximal_ideal $ (ring_hom.ker_is_maximal_of_surjective φ) hφ
 
+lemma residue_map_is_local_ring_hom :
```

Comment:
I think the name `is_local_ring_hom_residue` more closely follows the usual mathlib style.